copier.yml

```diff
@@ -20,14 +20,34 @@ package_description:
   multiline: true
   help: "Please provide a short description for the package."
 
+
+
+# This is the quick exit option for the template. It defaults to
+# basic package setup (and we will add basic tests and docs)
 use_default:
   type: str
-  help: "Do you want to skip all remaining questions and simply use the provided default values?"
+  help: "Do you want to skip all remaining questions and use the default values?"
   choices:
     "No, I want to fully customize the template.": "no"
     "Yes, with all the bells and whistles.": "full"
-    "Yes, but with a minimal setup.": "minimal"
-  default: "no"
+    "Yes, but with a minimal setup (package, tests and docs only).": "minimal"
+  default: "minimal"
+
+documentation:
+  when: "{{ use_default == 'no' }}"
+  type: str
+  help: "Do you want to include documentation for your project and which framework do you want to use?"
+  choices:
+    "Sphinx (https://pydata-sphinx-theme.readthedocs.io/en/stable/index.html)": sphinx
+    "mkdocs-material (https://squidfunk.github.io/mkdocs-material)": mkdocs
+    "I don't need documentation for my project.": none
+  default: "sphinx"
+
+use_rtd:
```

Comment:
RTD is only needed if the user has some sort of CI setup or cloud-based hosting and if they want to use rtd. this makes it optional. it was being added to the base minimal template